<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_19">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Upgrade</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Upgrade Name: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="upgrade_name_line_edit">
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Upgrade Slot(s): </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="upgrade_slot_line_edit">
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Faction(s):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="faction_line_edit">
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Ship(s):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ship_line_edit">
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Attack(s):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="attacks_line_edit">
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Arc Type(s): </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="arc_type_line_edit">
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Agility: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="arc_type_line_edit_2">
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
